pulsar-common/src/main/java/org/apache/pulsar/policies/data/loadbalancer/LocalBrokerData.java

```diff
@@ -250,10 +250,10 @@ public String printResourceUsage() {
                 bandwidthOut.percentUsage());
     }
 
-    public double getMaxResourceUsageWithWeight(final double cpuWeight, final double memoryWeight,
+    public double getMaxResourceUsageWithWeight(final double cpuWeight,
                                                 final double directMemoryWeight, final double bandwidthInWeight,
                                                 final double bandwidthOutWeight) {
```

Comment:
This is a breaking change to a public method users would likely call if they have custom `LoadSheddingStrategy` implementations. If we move forward removing the memory weight, I think we should first deprecate this method and then remove it later.

Note that we can still change the Pulsar implementations without breaking this method.

Reply:
I will deprecate this method first